packages/admin/admin/src/common/buttons/feedback/FeedbackButton.tsx

```diff
@@ -0,0 +1,165 @@
+import { ThreeDotSaving } from "@comet/admin-icons";
+import { LoadingButton, LoadingButtonProps } from "@mui/lab";
+import { ButtonClassKey, ComponentsOverrides } from "@mui/material";
+import { Theme, useThemeProps } from "@mui/material/styles";
+import * as React from "react";
+
+import { createComponentSlot } from "../../../helpers/createComponentSlot";
+import { ThemedComponentBaseProps } from "../../../helpers/ThemedComponentBaseProps";
+import { Tooltip as CometTooltip } from "../../Tooltip";
+
+export type FeedbackButtonClassKey = "idle" | "loading" | "success" | "error" | "tooltip" | ButtonClassKey;
+
+type OwnerState = { displayState: FeedbackButtonDisplayState };
+
+const Root = createComponentSlot(LoadingButton)<FeedbackButtonClassKey, OwnerState>({
+    componentName: "FeedbackButton",
+    slotName: "root",
+    classesResolver(ownerState) {
+        return [
+            ownerState.displayState === "idle" && "idle",
+            ownerState.displayState === "error" && "error",
+            ownerState.displayState === "success" && "success",
+            ownerState.displayState === "loading" && "loading",
+        ];
+    },
+})();
+
+const Tooltip = createComponentSlot(CometTooltip)<FeedbackButtonClassKey>({
+    componentName: "FeedbackButton",
+    slotName: "tooltip",
+})();
+
+export interface FeedbackButtonProps
+    extends ThemedComponentBaseProps<{
+            root: typeof LoadingButton;
+            tooltip: typeof CometTooltip;
+        }>,
+        LoadingButtonProps {
+    loading?: boolean;
+    hasErrors?: boolean;
+    startIcon?: React.ReactNode;
+    endIcon?: React.ReactNode;
+    tooltipSuccessMessage?: React.ReactNode;
+    tooltipErrorMessage?: React.ReactNode;
```

Comment:
What happens if I don't provide this values (since there are no defaults)?

Reply:
Very good point - thank you!

I set the defaultMessages to "Success" and "Error" - do you think this is enough? In the design it says "Saving was successful" and "Error while copying". But I think, it doesn't make sense to specify the message, as the button can be used for different use cases...